### PR TITLE
Add Microwave translations and update some Dutch ones

### PR DESCRIPTION
**Pull request by taakberg**
I noticed the `Cooking.Oven.Program.Microwave.*` and `Cooking.Oven.Program.Cleaning.*` strings were missing after buying a new oven. Additionally, the Dutch translations were a bit off and not in line with the Home Connect app, so I updated them.

### CODE REVIEW

File: custom_components/home_connect_alt/translations/nl.json [hunk 1]
```diff
@@ -335,26 +335,33 @@
           "Cooking.Common.Program.Hood.Venting": "Ventilatie",
           "Cooking.Hob.Program.PowerLevelMode": "Power-Level Mode",
           "Cooking.Oven.Program.Cleaning.Ecolysis": "Reinigingsfunctie",
-          "Cooking.Oven.Program.HeatingMode.BottomHeating": "Onder Verwarmen",
+          "Cooking.Oven.Program.HeatingMode.BottomHeating": "Onderverwarming",
```
Comment (golles) [inline custom_components/home_connect_alt/translations/nl.json:338]:
```suggestion
          "Cooking.Oven.Program.HeatingMode.BottomHeating": "Onderwarmte",
```

File: custom_components/home_connect_alt/translations/nl.json [hunk 2]
```diff
@@ -335,26 +335,33 @@
...
           "Cooking.Oven.Program.HeatingMode.Defrost": "Ontdooien",
           "Cooking.Oven.Program.HeatingMode.Desiccation": "Uitdroging",
-          "Cooking.Oven.Program.HeatingMode.FrozenHeatupSpecial": "Bevroren Opwarmen",
-          "Cooking.Oven.Program.HeatingMode.GrillLargeArea": "Grill, Groot Gebied",
-          "Cooking.Oven.Program.HeatingMode.HotAir": "Hete Lucht",
-          "Cooking.Oven.Program.HeatingMode.HotAirEco": "Hete Lucht Eco",
-          "Cooking.Oven.Program.HeatingMode.HotAirGentle": "Zachte Hete Lucht",
-          "Cooking.Oven.Program.HeatingMode.HotAirGrilling": "Hete Lucht Grillen",
+          "Cooking.Oven.Program.HeatingMode.FrozenHeatupSpecial": "Bevroren opwarmen",
+          "Cooking.Oven.Program.HeatingMode.GrillLargeArea": "Grill, groot gebied",
```
Comment (golles) [inline custom_components/home_connect_alt/translations/nl.json:342]:
```suggestion
          "Cooking.Oven.Program.HeatingMode.GrillLargeArea": "Grill, groot",
```

File: custom_components/home_connect_alt/translations/nl.json [hunk 3]
```diff
@@ -335,26 +335,33 @@
...
+          "Cooking.Oven.Program.HeatingMode.HotAirGrilling": "Hetelucht grillen",
           "Cooking.Oven.Program.HeatingMode.IntensiveHeat": "Intensieve Hitte",
           "Cooking.Oven.Program.HeatingMode.KeepWarm": "Warmhouden",
-          "Cooking.Oven.Program.HeatingMode.LetRest": "Laten Rusten",
-          "Cooking.Oven.Program.HeatingMode.PizzaSetting": "Pizza Instelling",
+          "Cooking.Oven.Program.HeatingMode.LetRest": "Laten rusten",
+          "Cooking.Oven.Program.HeatingMode.PizzaSetting": "Pizzastand",
           "Cooking.Oven.Program.HeatingMode.PreHeating": "Voorverwarmen",
-          "Cooking.Oven.Program.HeatingMode.PreheatOvenware": "Voorverwarmen Servies",
+          "Cooking.Oven.Program.HeatingMode.PreheatOvenware": "Voorverwarmen van ovenschalen",
```
Comment (golles) [inline custom_components/home_connect_alt/translations/nl.json:352]:
```suggestion
          "Cooking.Oven.Program.HeatingMode.PreheatOvenware": "Servies voorverwarmen",
```

File: custom_components/home_connect_alt/translations/nl.json [hunk 4]
```diff
@@ -335,26 +335,33 @@
...
           "Cooking.Oven.Program.HeatingMode.Proof": "Rijzen",
-          "Cooking.Oven.Program.HeatingMode.SlowCook": "Langzaam Koken",
-          "Cooking.Oven.Program.HeatingMode.TopBottomHeating": "Onder Boven Verwarmen",
-          "Cooking.Oven.Program.HeatingMode.TopBottomHeatingEco": "Onder Boven Verwarmen Eco",
+          "Cooking.Oven.Program.HeatingMode.SlowCook": "Langzaam garen",
+          "Cooking.Oven.Program.HeatingMode.TopBottomHeating": "Boven/onderverwarming",
```
Comment (golles) [inline custom_components/home_connect_alt/translations/nl.json:355]:
```suggestion
          "Cooking.Oven.Program.HeatingMode.TopBottomHeating": "Boven- en onderwarmte",
```

File: custom_components/home_connect_alt/translations/nl.json [hunk 5]
```diff
@@ -335,26 +335,33 @@
...
+          "Cooking.Oven.Program.HeatingMode.TopBottomHeatingEco": "Boven/onderverwarming eco",
```
Comment (golles) [inline custom_components/home_connect_alt/translations/nl.json:356]:
```suggestion
          "Cooking.Oven.Program.HeatingMode.TopBottomHeatingEco": "Boven- en onderwarmte Eco",
```

**Review by golles — COMMENTED** [review]
Just got my new oven. My oven and manual have these names

**Comment by taakberg** [comment]
@golles Thanks for checking! I copied the program names from the Home Connect app. Apparently, the oven itself uses different names. I just checked my oven, and the program names seem to mostly correspond with what you're suggesting. Only "Servies voorverwarmen" is called "Vorm voorverwarmen" on my oven, but I guess any one of these is fine.

I'll update my PR soon with the changes you've suggested.

**Comment by taakberg** [comment]
I also just found out that the oven has individual programs for "Gerechten" (dishes). Each individual choice has it's own translation string like `Cooking.Oven.Program.Dish.Recommendation.Conv.SimpleSpongeCake`. But I won't add all of those, there must be hundreds of them 😅

**Comment by golles** [comment]
> @golles Thanks for checking! I copied the program names from the Home Connect app. Apparently, the oven itself uses different names. I just checked my oven, and the program names seem to mostly correspond with what you're suggesting. Only "Servies voorverwarmen" is called "Vorm voorverwarmen" on my oven, and "Boven- en onderwarmte eco" is called "Milde boven- en onderwarmte" but I guess any one of these is fine.
> 
> 
> 
> I'll update my PR soon with the changes you've suggested.

Oh yes indeed, my oven calles it also "Vorm voorverwarmen", what I said earlier was from the manual. 

👍

**Comment by taakberg** [comment]
@ekutner Any chance you could merge this PR soon? If there's any feedback or anything I can do to help speed up the review, please let me know 🙂

**Review by ekutner — COMMENTED** [review]
There seem to be conflicts in this branch. Can you fix them please, so I can merge them?

**Comment by taakberg** [comment]
@ekutner Done!

**Review by ekutner — COMMENTED** [review]
Thank you! merged

**Pull request merged**